teamB/JeongWooJin/controllers/postController.js

```diff
@@ -0,0 +1,58 @@
+const postService = require('../services/postService');
+
+const enrollPost = async (req, res) => {
+  try {
+    const { title, content, user_id } = req.body;
+    
+    if ( !title || !content || !user_id ) {
+      return res.status(400).json({ message: 'KEY_ERROR' });
+    }
+   
+    await postService.enrollPost( title, content, user_id );
+    return res.status(201).json({
+      message: 'POST_SUCCESS',
+    });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const postLookup = async (req, res) => {
+  try {
+    const result = await postService.postLookup();
+    return res.status(201).json({data : result});
```

Comment:
상태코드 201은 데이터가 생성 되었을 때 사용합니다. 
일반적인 성공 케이스에서는 200을 사용합니다.
200, 201 이외에도 상황마다 적절하게 사용되는 상태코드들이 있습니다. 한 번 조사해 보면 좋을 것 같습니다.
이 부분 말고도 수정이 필요한 부분이 있으면 찾아서 수정해 주세요.
```suggestion
    return res.status(200).json({data : result});
```